app/src/main/res/layout/revision_card_activity.xml

```diff
@@ -29,19 +29,29 @@
         app:navigationContentDescription="@string/navigate_up"
         app:navigationIcon="?attr/homeAsUpIndicator">
 
+        <ImageButton
+          android:id="@+id/action_bottom_sheet_options_menu"
+          style="?android:actionOverflowButtonStyle"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_gravity="end"
+          android:minWidth="48dp"
+          android:minHeight="48dp"
+          android:scaleType="center" />
+
         <asia.ivity.android.marqueeview.MarqueeView
           android:id="@+id/revision_card_marquee_view"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
           app:animationDelay="500"
           app:animationSpeed="10">
 
-          <TextView
-            android:id="@+id/revision_card_toolbar_title"
-            style="@style/MarqueeToolbarTitleTextView"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_marginEnd="20dp" />
+        <TextView
+          android:id="@+id/revision_card_toolbar_title"
+          style="@style/MarqueeToolbarTitleTextView"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginEnd="20dp" />
```

Comment:
I think you accidentally reverted the spacing adjustment here--please fix.

Reply:
Done